frameworks/remix-run/application/src/routes/$langstore.order.invoice.$id[.pdf].tsx

```diff
@@ -0,0 +1,36 @@
+import { LoaderFunction } from '@remix-run/node';
+import { Response } from '@remix-run/web-fetch';
+import ReactPDF from '@react-pdf/renderer';
+import { Invoice } from '~/ui/components/pdf/order-invoice';
+import { createOrderFetcher } from '@crystallize/js-api-client';
+import { getStoreFront } from '~/use-cases/storefront.server';
+import { getContext } from '~/use-cases/http/utils';
+
+export const loader: LoaderFunction = async ({ params, request }) => {
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`